cfpb/consumerfinance.gov
# Form explainers: heading field and Wagtail usability updates

Updates form explainer module labels and helper text based on feedback from testing with internal Wagtail users. Also adds an optional heading field to the form explainer module to avoid the additional `.block` spacing that's automatically applied when the heading is added in a separate block.

## Additions

- Optional heading field in form explainer module to fix spacing issue for Prepaid use case


## Changes

- More descriptive labels and helper text in form explainer module
- Fixes element references in form explainer custom CSS so coordinate inputs are displayed inline in the Wagtail admin


## How to test this PR

1. Check out branch and run migration
2. Create a form explainer page and add a form explainer module
3. Verify explainer module styling and text appear as shown in admin screenshots below
4. Add a heading to the explainer module and verify that it is displayed as shown in output screenshot below


## Screenshots

### Admin styling and text updates

| <img width="1244" alt="Screen Shot 2022-09-02 at 11 24 03 AM" src="https://user-images.githubusercontent.com/778171/190027882-980eddd8-753a-4cf4-82e1-e53c07a09226.png"> |
| -- |


### Admin heading option
| <img width="1222" alt="Screen Shot 2022-09-12 at 10 07 27 AM" src="https://user-images.githubusercontent.com/778171/190027830-77d98f06-f2fd-4a0d-b35b-3e5884b06070.png"> |
| -- |

### Explainer heading output ("Sample prepaid card disclosure")
| <img width="1268" alt="Screen Shot 2022-09-12 at 9 53 48 AM" src="https://user-images.githubusercontent.com/778171/190027900-70a0cc65-6af4-45b1-b478-1f4886c2c37c.png"> |
| -- |


## Checklist



  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance
